rdrf/registry/patients/models.py

```diff
@@ -756,19 +758,38 @@ def hard_delete_clinicaldata(self, patient_id):
         for cd in clinicaldata_models:
             cd.delete()
 
-    def set_consent(self, consent_model, answer=True, commit=True):
+    def notify_consent_change(self, consent_value):
```

Comment:
This should change to be invoked only once per consent form save, not for each value change.

We should also make it a more general notification and not tie it notify only the patient's clinician.
That way, it could be used to notify other people as well. All they will have to do is to configure email notifications for them.

Reply:
Updated. For now it's still tied to the clinician, will change once we'll have more information about other roles to receive the notification.